### PR TITLE
Feature/architecture

### PR DESCRIPTION
**Pull request by yangdav9**
Please merge linux_sql/assets/linuxSqlArchitecture.jpg
Feature Branch PR URL (architecture & demo): https://www.notion.so/Architecture-Demo-366572b79590438fa9388c88c40db74d

Please merge linux_sql/README.md as well
Feature Branch PR URL (README.md): https://www.notion.so/README-md-829825719d72446f9439fb36fcc07b16

### CODE REVIEW

**Comment by yangdav9** [comment]
Please merge llinux_sql/README.md as well
Feature Branch PR URL (README.md): https://www.notion.so/README-md-829825719d72446f9439fb36fcc07b16

File: linux_sql/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+internally via switch and records the hardware specifications and storage details of the servers and stores 
+them into a database via PostgreSQL. The infrastructure team is then able to query information regarding server 
+host id by CPU number and sort by their memory size, average memory usage of each host and display when servers 
+are down.
+
+# 2) Architecture and Design #
+![my_architecture](./assets/linuxSqlArchitecture.jpg)
+
+# 3) Explanation of files #
+###### 1) ./scripts/host_info.sh ######
```
Comment (Jarvis-deprecated-account) [inline linux_sql/README.md:12]: ###### is a title
please use either inline code or block code format in this section

**Review by ghost — CHANGES_REQUESTED** [review]

File: linux_sql/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Contains create_table definitions for host_info and host_usage tables
+###### 5) ./sql/queries.sql ######
+Contains queries to return analysis of tables. Queries include:
+	1. Grouping hosts by hardware information
+	2. Printing the average memory use of all hosts
+	3. Detects whether node failure occurs in any of the hosts
+
+
+# 4) Script Usage #
+./scripts/psql_docker.sh (*start|stop*) [*password|*] <br />
```
Comment (Jarvis-deprecated-account) [inline linux_sql/README.md:28]: again, inline code format or block code format.

File: linux_sql/README.md (hunk 1 above)
Comment (Jarvis-deprecated-account) [reply linux_sql/README.md:12]: markdown bold format `**bold**`

Comment (yangdav9) [reply linux_sql/README.md:12]: Edited with inline code

File: linux_sql/README.md (hunk 2 above)
Comment (yangdav9) [reply linux_sql/README.md:28]: Edited with inline code

**Pull request merged**